packages/manager/apps/pci-public-ip/src/components/list/FailoverIP.component.tsx

```diff
@@ -300,7 +300,7 @@ export default function FailoverIPComponent({
       )}
 
       {!isLoading && !error && (
-        <div className="mt-8 overflow-x-auto">
```

Comment:
if we remove this the overflow will be applied for all the page not only the datagrid